scripts/create-pr-645-646.sh

```
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -e
+
+# Get repository root dynamically (cross-platform compatible)
+if command -v git >/dev/null 2>&1 && git rev-parse --show-toplevel >/dev/null 2>&1; then
+  REPO_ROOT=$(git rev-parse --show-toplevel)
+elif [ -n "$BASH_SOURCE" ]; then
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)